Dockerfile

```diff
@@ -1,9 +1,12 @@
 FROM golang:1.13 as builder
 
+# Install packr
+RUN go get -u github.com/gobuffalo/packr/packr
```

Comment:
Can we use [v2](https://github.com/gobuffalo/packr/blob/master/v2/README.md) instead?